migrations/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: init install
+
+install:
+	npm install -g pg-diff-cli
+	go install github.com/pressly/goose/v3/cmd/goose@latest
+
```

Comment:
I'm in general not a fan of having this in makefiles. IMO the setup of the tools should not be the responsibility of the system, the system should just tell you what you are missing and suggest a copyable solution.